Use opacity only for disabled states of checkbox, radio (take 2 🎬)

dancormier opened the pull request:
This PR addresses https://github.com/StackExchange/Stacks/issues/672 by removing color overrides for checkbox and radio inputs to solely use 50% opacity when disabled.

Take 1: https://github.com/StackExchange/Stacks/pull/674

dancormier committed:
Remove checkbox, radio disabled colors

https://github.com/StackExchange/Stacks/issues/672

Take 1: https://github.com/StackExchange/Stacks/pull/674

aaronshekey commented:
Looking good! Didn't realize we already rendered these at `50%` and _then_ recolored. Nice!

I think it'll be worth it for future testing to add the checked (but disabled) state in this example: 
![image](https://user-images.githubusercontent.com/1369864/120374482-33f32380-c2df-11eb-94b9-8b69771182ae.png)

Maybe we just double this row and have the second row both `checked`, leaving the copy & paste markup alone.

Like so:
![image](https://user-images.githubusercontent.com/1369864/120374938-cbf10d00-c2df-11eb-8728-31fa301b126d.png)

dancormier committed:
update checkbox doc with checked checkboxes

addresses https://github.com/StackExchange/Stacks/pull/675#issuecomment-852360062

dancormier committed:
Update radio doc with checked checkboxes

addresses https://github.com/StackExchange/Stacks/pull/675#issuecomment-852360062
